components/shared/AccountPicker.tsx

```diff
@@ -0,0 +1,174 @@
+import React, { useState, useEffect, useRef } from "react";
+import { useCENNZExtension } from "@/providers/CENNZExtensionProvider";
+import { css } from "@emotion/react";
+import AccountIdenticon from "@/components/shared/AccountIdenticon";
+import { useCENNZWallet } from "@/providers/CENNZWalletProvider";
+import { useBridge } from "@/providers/BridgeProvider";
+import Jazzicon, { jsNumberForAddress } from "react-jazzicon";
+
+const AccountPicker: React.FC<{
+	updateSelectedAccount: Function;
+	topText: string;
+	cennznet: boolean;
```

Comment:
Could be good to use the SupportedChain type here rather a boolean

Reply:
sweet should be fixed now